.github/workflows/build-firmware.yaml

```diff
@@ -133,6 +133,27 @@ jobs:
         name: console ${{matrix.build-target}} junit
         path: ./java_console/build/*.txt
 
+    - name: Upload build bin
+      if: ${{ github.event_name != 'push' || github.ref != 'refs/heads/master' }}
+      uses: actions/upload-artifact@v2
+      with:
+        name: rusefi_${{matrix.build-target}}.bin
+        path: ./firmware/deliver/rusefi_${{matrix.build-target}}.bin
+
+    - name: Upload build hex
+      if: ${{ github.event_name != 'push' || github.ref != 'refs/heads/master' }}
```

Comment:
copy-paste? can we squash three path uploads int one?

i was thinking to upload the whole bundle as is, not just individual files since we seem to be building the bundle? i could be wrong you have better context

Reply:
Nope, not building the bundles on a PR. In order to build the bundles, we would have to build simulator and console as well, which we moved away from to keep from making firmware build fail when console or simulator builds failed.